test/ParseableInterface/ModuleCache/module-cache-init.swift

```diff
@@ -24,7 +24,7 @@
 // RUN: test -f %t/modulecache/LeafModule-*.swiftmodule
 // RUN: llvm-bcanalyzer -dump %t/modulecache/LeafModule-*.swiftmodule | %FileCheck %s -check-prefix=CHECK-LEAFMODULE
 // CHECK-LEAFMODULE: {{MODULE_NAME.*blob data = 'LeafModule'}}
-// CHECK-LEAFMODULE: {{FILE_DEPENDENCY.*Swift.swiftmodule([/\\].+[.]swiftmodule)?'}}
+// CHECK-LEAFMODULE-NOT: {{FILE_DEPENDENCY.*Swift.swiftmodule([/\\].+[.]swiftmodule)?'}}
```

Comment:
:-/ If Swift.swiftmodule started showing up after LeafModule.swiftmodule, we wouldn't catch it this way. I think you'll want something like CHECK-DEPENDS-NEGATIVE below.

Reply:
Fixed!